opendps/tick.c

```diff
@@ -73,6 +78,8 @@ uint64_t get_ticks(void)
   */
 void sys_tick_handler(void)
 {
-    tick_ms++;
+    ticks_lower++;
+    if (ticks_lower == 0) // If an overflow has occured
+        ticks_upper++;
 }
```

Comment:
This assumes that `get_ticks()` is never called from an interrupt with a higher priority

Reply:
Ah, clever! Nice to learn something new every now and then 🎉